### ISSUE TITLE
Certus Quartz Seed, Nether Quartz Seed despawn due to WYML

### ISSUE THREAD

**Issue by Aluminus** [MechOrigin/NTC2#120]
The WYML config contains a blacklist for the item despawn mechanic, but it is empty, so it will apply its default item despawn time (5 mins.) to ALL items. Adding the quartz seed items to the ITEM_DESPAWN_DENYLIST fixes this issue. Without this fix, progressing in AE2 is virtually impossible.

Example fix that worked for me (wyml.json):
`"ITEM_DESPAWN_DENYLIST":
[
	"appliedenergistics2:nether_quartz_seed",
	"appliedenergistics2:certus_crystal_seed",
	"appliedenergistics2:fluix_crystal_seed"
]`

**Comment by MechOrigin** [comment]
This will be fixed in version 1.5.1! Thank you <3

**Comment by JL2210** [comment]
They despawn after they finish growing, too. Maybe add the purified crystals to the blacklist?
```
	"ITEM_DESPAWN_DENYLIST": [
		"appliedenergistics2:nether_quartz_seed",
		"appliedenergistics2:purified_nether_quartz_crystal",
		"appliedenergistics2:certus_crystal_seed",
		"appliedenergistics2:purified_certus_quartz_crystal",
		"appliedenergistics2:fluix_crystal_seed",
		"appliedenergistics2:purified_fluix_crystal"
	],
```
